app/Models/Competition.php

```diff
@@ -12,14 +11,14 @@ class Competition extends Model
      *
      * @var array
      */
-    protected $fillable = ['campaign_id', 'campaign_run_id', 'start_date', 'end_date'];
+    protected $fillable = ['contest_id', 'competition_start_date', 'competition_end_date'];
```

Comment:
A competition just references a the contest that it is apart of. This is fillable so we can create competitions and set the contest_id for them.